Sources/BraveWallet/Crypto/Transaction Confirmations/TransactionStatusView.swift

```diff
@@ -0,0 +1,151 @@
+// Copyright 2023 The Brave Authors. All rights reserved.
+// This Source Code Form is subject to the terms of the Mozilla Public
+// License, v. 2.0. If a copy of the MPL was not distributed with this
+// file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+import SwiftUI
+import BraveCore
+import BigNumber
+import Strings
+import DesignSystem
+
+struct TransactionStatusView: View {
+  @ObservedObject var confirmationStore: TransactionConfirmationStore
+  let networkStore: NetworkStore
+  @Binding var transactionDetails: TransactionDetailsStore?
+ 
+  let onDismiss: () -> Void
+  
+  @Environment(\.openWalletURLAction) private var openWalletURL
+  
+  @ViewBuilder private var signedOrSubmittedTxView: some View {
+    GeometryReader { geometry in
+      ScrollView(.vertical) {
+        VStack(spacing: 10) {
+          Image("tx-submitted", bundle: .module)
+          Text(confirmationStore.activeTxStatus == .signed ? Strings.Wallet.signedTransactionTitle : Strings.Wallet.submittedTransactionTitle)
+            .font(.title3.bold())
+            .foregroundColor(Color(.braveLabel))
+            .multilineTextAlignment(.center)
+            .padding(.top, 10)
+          Text(confirmationStore.activeTxStatus == .signed ? Strings.Wallet.signedTransactionDescription : Strings.Wallet.submittedTransactionDescription)
+            .font(.subheadline)
+            .foregroundColor(Color(.secondaryBraveLabel))
+            .multilineTextAlignment(.center)
+          Button {
+            onDismiss()
+          } label: {
+            Text(Strings.OKString)
+              .padding(.horizontal, 8)
+          }
```

Comment:
nit:
```suggestion
          Button(action: onDismiss) {
            Text(Strings.OKString)
              .padding(.horizontal, 8)
          }
```

Reply:
updated in [8d59b95](https://github.com/brave/brave-ios/pull/6960/commits/8d59b95439fdfcd0da5f733c63d51fea80b2a899)